plugins/figura_format/figura_format.js

```diff
@@ -147,266 +115,339 @@
+						],
+						form: {
+							"all_animations": {
+								type: 'checkbox',
+								label: 'Bake all Animations?',
+								description: "This Action will normally only bake the selected Animation. Do you want to bake all Animations in one swoop?",
+								value: false,
+								full_width: false
+							}
+						},
+						onConfirm() {
+							const animations = this.getFormResult().all_animations ? Animator.animations : [Animation.selected]
+							for (const animation of animations) {
+								let animators = animation.animators
 
-									// Inverse Kinematics
-									let ik_samples = animation.sampleIK();
-									for (let uuid in ik_samples) {
-										//let group = OutlinerNode.uuids[uuid];
-										ik_samples[uuid].forEach((rotation, i) => {
-											let timecode = i / animation.snapping
-											let kf = getOrMakeKeyframe(animators[uuid], 'rotation', timecode, animation.snapping)
-											kf.set('x', rotation.array[0])
-											kf.set('y', rotation.array[1])
-											kf.set('z', rotation.array[2])
-										})
-									}
+								// Inverse Kinematics
+								let ik_samples = animation.sampleIK();
+								for (let uuid in ik_samples) {
+									//let group = OutlinerNode.uuids[uuid];
+									ik_samples[uuid].forEach((rotation, i) => {
+										let timecode = i / animation.snapping
+										let kf = getOrMakeKeyframe(animators[uuid], 'rotation', timecode, animation.snapping)
+										kf.set('x', rotation.array[0])
+										kf.set('y', rotation.array[1])
+										kf.set('z', rotation.array[2])
+									})
 								}
 							}
-						}).show()
-					}
-				}), '#edit')
-
-
-			// Remove the Texture Render Mode field from the Right Click context menu.
-			Texture.prototype.menu.structure.find(v => v.name == 'menu.texture.render_mode').condition = () => Format !== format
-			// In the Texture Properties Dialog specifically, remove the Render Mode field
-			let DialogBuild = Dialog.prototype.build
-			Dialog.prototype.build = function () {
-				if (Format === format && this.id == 'texture_edit') delete this.form.render_mode
-				DialogBuild.call(this)
-			}
-
-			// Prevents the Timeline from erroring when Sound and Particle channels are removed
-			let displayFrame = EffectAnimator.prototype.displayFrame
-			EffectAnimator.prototype.displayFrame = function () {
-				if (Format === format) return
-				displayFrame.call(this)
-			}
+						}
+					}).show()
+				}
+			})
+			const cycleVertexOrder = new Action('figura_cycle_vertex_order', {
+				name: 'Cycle Vertex Order',
+				icon: 'fa-sync',
+				category: 'edit',
+				condition: { modes: ['edit'], features: ['meshes'], method: () => Format === format && (Mesh.selected[0] && Mesh.selected[0].getSelectedFaces().length) },
+				click() {
+					Undo.initEdit({ elements: Mesh.selected });
+					Mesh.selected.forEach(mesh => {
+						for (const face of mesh.faces) {
```

Comment:
Mesh.faces is not an array, it's an object, so the function fails to run.

Reply:
welp, this action was removed. Resolved